projects/sbb-esta/angular-business/src/lib/contextmenu/contextmenu/contextmenu.component.scss

```diff
@@ -0,0 +1,87 @@
+.mask-right {
+  position: absolute;
+  left: 24px;
+  width: 1px;
+  height: 23px;
+  background-color: #666;
```

Comment:
Use predefined colors from _colors.scss
Use `@import 'common';` to have them available
This also applies to the colors below